modules/widgets/wordpress-post-widget.php

```diff
@@ -926,7 +926,12 @@ public function form( $instance ) {
 				<?php _e( "Enter a WordPress.com or Jetpack WordPress site URL.", 'jetpack' ); ?>
 			</i>
 			<?php
-			if ( empty( $url ) ) {
+			/**
+			 * Show an error if the URL field was left empty.
+			 *
+			 * The error is shown only when the widget was already saved.
+			 */
+			if ( empty( $url ) && ! preg_match( '/__i__|%i%/', $this->id ) ) {
```

Comment:
I am not entirely sure if this is the best way to check if the widget has been already saved or not.

Is there a better way for that?